keras/legacy/interfaces.py

```diff
@@ -88,3 +89,16 @@ def raise_duplicate_arg_error(old_arg, new_arg):
 legacy_gaussiannoise_support = generate_legacy_interface(
     allowed_positional_args=['stddev'],
     conversions=[('sigma', 'stddev')])
+
+legacy_recurrent_support = generate_legacy_interface(
+    allowed_positional_args=['units'],
+    conversions=[('output_dim', 'units'),
+                 ('init', 'kernel_initializer'),
+                 ('inner_init', 'recurrent_initializer'),
+                 ('inner_activation', 'recurrent_activation'),
+                 ('forget_bias_init', 'bias_initializer'),
```

Comment:
This is not the same argument. The proper behavior here would be:
- if `forget_bias_init` is set to `"one"`, set `unit_forget_bias` to True
- if set to `zero`, set `unit_forget_bias` to False
- else ignore the argument, and in this case, raise a warning specifying the argument was ignored.

You will need custom code to do this.